### PR TITLE
Fix940 Added topics: bank-account

### LINKED ISSUE #940: bank-account: add topics
As part of the update to the exercism website, we want to add topics to each exercise. This should be topics that will most likely be relevant to the exercise even though you might be able to solve it without encountering that topic. There is a [list of topics](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) used across all exercism tracks but you can add track specific ones as well if needed.

See issue #523 for a discussion about adding topics. There are also some exercises (e.g. hello-world and two-fer) that already have topics so you can look at them as examples. The topics need to be added to the entry for the relevant exercise in `config.json`.

Please comment on this issue or reference it in your pull request (using #issue-number) if you're going to be working on it. If you put "fixes #issue-number" in the pull request description then it will also automatically close this issue when the pull request gets merged :)

### PR DESCRIPTION
**Pull request by yashasingh**
Added topics to the question Bank Account. This PR fixes #940 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -737,7 +742,14 @@
...
       "difficulty": 6,
       "slug": "bank-account",
-      "topics": null,
+      "topics": [
+      	"floating_point_numbers",
+      	"files",
+      	"mathematics",
+      	"logic",
+        "object_oriented_programming",
+      	"variables"
```
Comment (FridaTveit) [inline config.json]: Thanks for tackling this :) I'm not sure I agree with your suggestions. `floating_point_numbers` and `files` don't appear to be used in this exercise at all. `mathematics`, `logic`, `object_oriented_programming` and `variables` are all used but I wouldn't say they are very important topics. How do you feel they are relevant to the exercise?

I would add `exception_handling` since they tests expect an exception to be thrown, and `concurrency` since that is a very important part of the exercise. You could maybe also add `integers`. What do you think? :)

There are some indentation problems. Each topic should be indented two spaces after `topics`, like `object_oriented_programming`.

Comment (yashasingh) [reply config.json]: Alright, I'll come up with more accurate topics and update the PR

**Comment by yashasingh** [comment]
@FridaTveit I've made the required changes. Please review.

File: config.json [hunk 2]
```diff
@@ -752,7 +752,12 @@
       "core": true,
       "difficulty": 6,
       "slug": "bank-account",
-      "topics": null,
+      "topics": [
+        "arrays",
```
Comment (FridaTveit) [inline config.json]: Why did you add this as a topic? I don't think you need to use arrays to solve this exercise. I know arrays are used in one of the tests, but not in a way that requires the person implementing the exercise to use arrays at all. Does that make sense? So I'm not sure arrays are important enough to this exercise to require it to be added as a topic. What do you think?

Comment (yashasingh) [reply config.json]: :+1:

Comment (yashasingh) [reply config.json]: I included this topic because it was being used at one the places. But, now just as you mentioned , I agree that arrays don't have anything to do with problem concept , so I'm removing it. Thank you.

**Comment by yashasingh** [comment]
@FridaTveit Please review.
